carbon-design-system/carbon-website
# docs(v11): update type tokens on style tabs

Closes #2511 

**Changed**
- Updated all component style tab typography tables to reflect the new type token names.

----

**Testing**
- Go to each component style tab typography table.
- Make sure the type token names are correct.
